shared/templates/sudo_defaults_option/bash.template

```diff
@@ -3,19 +3,35 @@
 # strategy = restrict
 # complexity = low
 # disruption = low
+{{% if VARIABLE_NAME %}}
+# Include source function library.
+. /usr/share/scap-security-guide/remediation_functions
 
+{{{ bash_instantiate_variables(VARIABLE_NAME) }}}
+{{% endif %}}
 if /usr/sbin/visudo -qcf /etc/sudoers; then
-    if ! grep -P "^[\s]*Defaults.*\b{{{ OPTION }}}\b.*$" /etc/sudoers; then
-        cp /etc/sudoers /etc/sudoers.bak
-        echo "Defaults {{{ OPTION }}}" >> /etc/sudoers
-        if /usr/sbin/visudo -qcf /etc/sudoers; then
-            rm -f /etc/sudoers.bak
-        else
-            echo "Fail to validate remediated /etc/sudoers, reverting to original file."
-            mv /etc/sudoers.bak /etc/sudoers
-            false
+    cp /etc/sudoers /etc/sudoers.bak
+    if ! grep -P "^[\s]*Defaults.*\b{{{ OPTION_REGEX }}}\b.*$" /etc/sudoers; then
+        # sudoers file doesn't define Option {{{ OPTION }}}
+        echo "Defaults {{{ OPTION_VALUE }}}" >> /etc/sudoers
+    {{%- if not VARIABLE_NAME %}}
+    fi
+    {{% else %}}
+    else
+        # sudoers file defines Option {{{ OPTION }}}, remediate if appropriate value is not set
+        if ! grep -P "^[\s]*Defaults.*\b{{{ OPTION_VALUE }}}\b.*$" /etc/sudoers; then
+            sed -Ei "s/(^[\s]*Defaults.*\b{{{ OPTION }}}=)\w+(\b.*$)/\1{{{ '${' ~ VARIABLE_NAME ~ '}' }}}\2/" /etc/sudoers
```

Comment:
The usage of `{{{ '${' ~ VARIABLE_NAME ~ '}' }}}` is the most meaningful way how to do this, good job!